### PR TITLE
docs: improve install, debug and config guide

### PR DESCRIPTION
**Pull request by ravikp7**
- add install, debug guide
- improve aws config guide

### CODE REVIEW

**Comment by ravikp7** [comment]
See `Using a Shared Config file` section [here](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-region.html)

File: README.md [hunk 1]
```diff
@@ -114,16 +114,30 @@ If the `req.egContext.lambda` object is not populated, a default event structure
...
 
 Example:
 
 ```
-AWS_SDK_LOAD_CONFIG=true node server.js
+AWS_SDK_LOAD_CONFIG=true npm start
+```
+---
+### Debugging:
+If you get `Internal Server Error` from plugin, see debug logs
```
Comment (XVincentX) [inline README.md]:
```suggestion
If you get `Internal Server Error` from plugin, you can see the extended logs by doing:
```

**Review by XVincentX — APPROVED** [review]
Nice! Looks ok to me!

**Comment by ravikp7** [comment]
Thanks! done with the suggested changes

**Pull request merged**